pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -237,6 +237,26 @@ def test_build_series(self):
 
         assert result == expected
 
+    def test_read_json_from_to_json_results(self):
+        # GH 32383
+        df = pd.DataFrame(
+            {
+                "_id": {"0": "0"},
+                "category": {"0": "Goods"},
+                "recommender_id": {"0": "3"},
+                "recommender_name_jp": {"0": "浦田"},
+                "recommender_name_en": {"0": "Urata"},
+                "name_jp": {"0": "博多人形（松尾吉将まつお よしまさ）"},
+                "name_en": {"0": "Hakata Dolls Matsuo"},
+            }
+        )
+        df_json = df.to_json()
+        df_dict = json.loads(df_json)
+        result = pd.read_json(df_json, dtype=False)
+        result.index = result.index.astype(str)
+        expected = pd.DataFrame.from_dict(df_dict)
```

Comment:
does this test need to use from_dict. IIUC from the OP https://github.com/pandas-dev/pandas/issues/32383#issuecomment-593252014 we are roundtripping to/from json and to_dict was part of a workaround.

Reply:
fixed

Reply:
can you create the expected explicitly, i.e not using from_dict.

Reply:
i am using original df now